code/game/jobs/job/engineering.dm

```diff
@@ -148,3 +148,33 @@
 	satchel = /obj/item/weapon/storage/backpack/satchel_eng
 	dufflebag = /obj/item/weapon/storage/backpack/duffel/eng
 	messengerbag = /obj/item/weapon/storage/backpack/messenger/engi
+
+/datum/job/engofficer
+	title = "Engineering Officer"
+	flag = ENGISEC
+	department = "Engineering"
+	department_flag = ENGSEC
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the chief engineer and head of security"
+	selection_color = "#dbba35"
+	economic_modifier = 4
+
+	minimum_character_age = 18
+
+	access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels, access_construction, access_atmospherics)
+	minimal_access = list(access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels, access_construction, access_atmospherics,)

```

Comment:
`minimal_access` has more access than `access`
`minimal_access` has a trailing `,`

We are running on `minimal_access` so make sure that they are either the same or `access` has actually more access than `minimal_access`